app/views/pages/privacy_policy.html.erb

```diff
@@ -0,0 +1,193 @@
+      we have employed to process your personal data on our behalf) and/or other organisations (with whom we need to share
+      your personal data for specific purposes).
+    </p>
+
+    <p class="govuk-body">
+      You must not misuse our service by knowingly introducing viruses, trojans, worms, logic bombs or other material
+      that is malicious or technologically harmful. You must not attempt to gain unauthorised access to our service,
+      the server on which our service is stored or any server, computer or database connected to our service. You must
+      not attack our site via a denial-of-service attack or a distributed denial-of service attack. Each of these acts
+      is a criminal offence. We will report any such offence to the relevant law enforcement authorities and co-operate
+      with them to determine your identity.
+    </p>
+
+    <p class="govuk-body">
+      Where we need to share your personal data with others, we ensure that this data sharing complies with data protection
+      legislation. We share specific types of personal data with dxw, who have been contracted by DfE to manage the delivery
+      and hosting of the digital service.
+    </p>
+
+    <p class="govuk-body">
+      The firm dxw use your data:
+    </p>
+
+    <ul class='govuk-list govuk-list--bullet'>
+      <li>
+        for security purposes: collecting part of the IP addresses of people visiting the Teaching Vacancies website
+        (for example, if we were receiving continuous direct denial of service attacks from an IP address range then we could
+        block it)
+      </li>
+      <li>
+        to enable users to use the service and receive updates
+      </li>
+    </ul>
+
+    <h2 class='govuk-heading-l'>
+      How long we will keep your personal data
+    </h2>
+
+    <p class="govuk-body">
+      Neither DfE nor its delivery partners will keep your personal data longer than we need it for the purpose(s) of
+      research, security and/or delivery of the service, and in any case not longer than 10 years. Your data will be deleted
+      when DfE and its delivery partners no longer need it for these purposes, or when 10 years have passed, whichever comes
+      first.
+    </p>
+
+    <p class="govuk-body">
+      Partial IP addresses will be deleted after 7 days unless their request caused an error, in which case they remain in
+      our logs for 30 days.
+    </p>
+
+    <h2 class='govuk-heading-l'>
+      Your data protection rights
+    </h2>
+
+    <p class="govuk-body">
+    You have the right:
+    </p>
+
+    <ul class='govuk-list govuk-list--bullet'>
+      <li>
+        to ask us for access to information about you that we hold
+      </li>
+      <li>
+        to have your personal data rectified, if it is inaccurate or incomplete
+      </li>
+      <li>
+        to ask us to delete your personal data unless there is a legal reason to continue to store and process it
```

Comment:
Do they have this right? Or does "prevention of repeat claims" count as a "legal reason"?

Reply:
I think it does, again I'd defer to a specialist in this field